go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	github.com/edaniels/golog v0.0.0-20230215213219-28954395e8d0 // indirect
 	github.com/edaniels/zeroconf v1.0.10 // indirect
 	github.com/goccy/go-json v0.10.2 // indirect
-	github.com/godbus/dbus/v5 v5.1.0 // indirect
+	github.com/godbus/dbus/v5 v5.1.1-0.20241109141230-b9236d654833 // indirect
```

Comment:
This is from a pending branch that adds a feature needed for the return type of the StateReason() interface that was added.